nano/lib/stats.cpp

```diff
@@ -456,6 +456,25 @@ void nano::stats::stop ()
 	stopped = true;
 }
 
+std::string nano::stats::to_string (std::string type)
```

Comment:
Wouldn't it be easier to use and less error prone if the parameter was an enum?

Reply:
In theory yes, in practice, I don't know. everywhere the stats class uses strings for these things.
So I just followed the theme started by the stats class.

Reply:
Okay, it's good to keep consistency in that case.

Reply:
I should clarify, the idea is to use this function from inside a debugger.
And I thought it made more sense to match the way you would use the RPC calls to get the stats.
It is easier to type "samples" in a debugger than to lookup the value of an enum.